cmds/core/kexec/kexec_linux.go

```diff
@@ -60,7 +59,7 @@ type options struct {
 
 func registerFlags() *options {
 	o := &options{}
-	flag.BoolVarP(&o.loadSyscall, "loadsyscall", "L", false, "Use the kexec load syscall (not file_load)")
+	flag.BoolVarP(&o.loadSyscall, "loadsyscall", "L", false, "Use the kexec_load syscall (not kexec_file_load)")
```

Comment:
I was using kexec_load in my original sketch, and then Ron changed away what is merged; and now it is changed back. It looks we don't have a preference / guideline in general

Reply:
IMO when referring to actual system calls, we use the system call name as used by Linux and the man pages. kexec_load is that name